packages/seal/src/Marshaller/Marshaller.php

```diff
@@ -319,6 +323,10 @@ private function unmarshallDateTimeField(string|int|array|null $value, Field\Dat
             return \date('c', $value);
         }
 
+        if (\is_string($value) && \str_ends_with($value, 'Z')) {
+            return \date('c', \strtotime($value));
+        }
```

Comment:
Solr removes the timezone and returns `Z` instead of `+01:00`. so we need to convert back it to our configured php timezone